.travis.yml

```diff
@@ -1,3 +1,8 @@
 language: node_js
 node_js:
-  - "6.9"
+  - "12.8.0"
+script: bash ./deploy.sh
```

Comment:
You'll need to create two separate travis builds here, one for running jest (`npm test`) and one for the deployment (`bash ./deploy.sh` which if you chmod it with execution permissions will probably work as just `./deploy.sh`, although you're way is probably "more correct").

Right now its only running the deploy and not executing the jest tests anymore. You can look how we do it in Inkwell if you want a reference, although that uses a build matrix which may be overkill here.

Reply:
Travis now runs one job for tests and one for docs.